doc/user-guides/authenticated-rl-with-jwt-and-k8s-authnz.md

```diff
@@ -54,45 +90,31 @@ curl -H 'Host: api.toystore.com' http://localhost:9080/toy -i
 
 It should return `200 OK`.
 
-**Note**: This only works out of the box on linux environments. If not on linux,
-you may need to forward ports
+> **Note**: If the command above fails to hit the Toy Store API on your environment, try forwarding requests to the service:
+>
+> ```sh
+> kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 2>&1 >/dev/null &
+> ```
 
-```bash
-kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 &
-```
-
-### ❹ Request the Kuadrant instance
-
-```sh
-kubectl -n kuadrant-system apply -f - <<EOF
----
-apiVersion: kuadrant.io/v1beta1
-kind: Kuadrant
-metadata:
-  name: kuadrant-sample
-spec: {}
-EOF
-```
-
-### ❺ Deploy Keycloak
+### ③ Deploy Keycloak
 
 Create the namesapce:
 
 ```sh
 kubectl create namespace keycloak
 ```
 
-Deploy Keycloak:
+Deploy Keycloak with a [bootstrap](https://github.com/kuadrant/authorino-examples#keycloak) realm, users, and clients:
 
 ```sh
 kubectl apply -n keycloak -f https://raw.githubusercontent.com/Kuadrant/authorino-examples/main/keycloak/keycloak-deploy.yaml
```

Comment:
I believe that having a shared repo that the different components can pull sample apps from may contribute to keeping consistent user guides and user stories across. Maybe we could make https://github.com/Kuadrant/authorino-examples more about Kuadrant as well? I wouldn't mind renaming it to "kuadrant-examples" if we think that would help.

Reply:
My request was not about ownership. In other words `authorino-examples` is ok for me. It was about, what if you change those resources to accommodate to new features or spec of Authorino? I fear that it would break the guides in the kuadrant operator. Anyway, raising this for awareness, not for this PR.